### PR TITLE
feat: update with latest bindings changes

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**

This PR adds the following changes:
- Updates the codeowners file with the latest bindings owners adding @mboss37, @rcoppen, @SrfHead, @dpwdec, @iancooper
- Removed `.keep` files where needed
- Adapted websocket JSON Schema files to use the 3.0.0 spec files
- Add sqs bindings
- Add jns bindings
- Adapted HTTP bindings
- Adapted anypointmq bindings
- Changed all specification extension patterns from `^x-[\\w\\d\\.\\-\\_]+$` to `^x-[\\w\\d\\.\\x2d_]+$`
- Changed all references to use v3

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -8,11 +8,14 @@
...
 
-/bindings/anypointmq/       @GeraldLoeffler
-/bindings/ibmmq/            @rcoppen
-/bindings/kafka/            @lbroudoux @dalelane
-/bindings/googlepubsub/     @whitlockjc
-/bindings/solace/           @damaru-inc @CameronRushton
-/bindings/*.json            @KhudaDad414
-/bindings/pulsar/           @VisualBean
\ No newline at end of file
+/bindings/*.json        @KhudaDad414
```
Comment (jonaslagoni) [inline CODEOWNERS:21]: @mboss37, @rcoppen, @SrfHead, @dpwdec, @iancooper the JSON Schema files for the bindings are as of v3 being validated alongside the specification, which is why the files have been moved from the bindings repo. @derberg will at some point send you an invite to join the spec-json-schemas repository as a maintainer so you still have full ownership of the bindings 🙂

**Review by dalelane — APPROVED** [review]
looks good once the new members are invited

**Comment by derberg** [comment]
merged instead of squash and merge

**Comment by mboss37** [comment]
I don't have any pending invite. So looks good for me

**Pull request merged**